test/cram/tap.t

```diff
@@ -96,3 +96,14 @@ Testing CRITERION_OUTPUTS
   ok - misc::passing 
   not ok - misc::failing 
     simple.c:4: Assertion failed: The expression 0 is false.
+
+
+
+  $ skip.c.bin --tap
+  TAP version 13
+  1..2
+  # Criterion v2.2.1
+  
+  # Running 2 tests from misc
+  ok - misc::skipping  # SKIP test was skipped
+  ok - misc::message  # SKIP test was skipped
```

Comment:
The TAP spec mandates having the skip message after SKIP here. (in other words it should be `Skips may take printf-like messages`)